aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferService.java

```diff
@@ -137,7 +142,8 @@ public int onStartCommand(Intent intent, int flags, int startId) {
          * b) An identifier for the ongoing notification c) Flag that determines if the notification
          * needs to be removed when the service is moved out of the foreground state.
          */
-        if (Build.VERSION.SDK_INT >= ANDROID_OREO) {
+        if (Build.VERSION.SDK_INT >= ANDROID_OREO && !hasNotificationShown) {
```

Comment:
do we need hasNotificationShown check here?

Reply:
I didn't want to re-trigger showing the same notification again, which could look glitchy if you were observing the drawer and the notification kept getting recreated